### PR TITLE
Make the Whole Card Clickable

### LINKED ISSUE #3151: [Make the icons clickable]
### Description

while visiting the LinkFree website I immediately clicked on icons but found out the Icons are not clickable but the text below is,
so I request that we could make those icons clickable because it comes by Natural Human-instinct to click them.

### Screenshots

Screenshot from 2023-01-11 16-33-04

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Sriparno08**
## Fixes Issue

Closes #3151

## Changes proposed

I have made the whole card clickable, instead of just the heading name.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3828"></a>

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -409,29 +409,29 @@ export default function Home({ total, today }) {
...
+                        <div className="-mt-6">
+                          <div>
+                            <span className="inline-flex items-center justify-center rounded-xl bg-indigo-500 p-3 shadow-lg">
+                              <feature.icon
+                                className="h-8 w-8 text-white"
+                                aria-hidden="true"
+                              />
+                            </span>
+                          </div>
+                          <h3 className="mt-8 text-lg font-semibold leading-8 tracking-tight hover:text-indigo-600">
```
Comment (Aadarsh805) [inline pages/index.js:426]: Make this `h3` hover effect to happen when hovered over the whole card

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: pages/index.js [hunk 2]
```diff
@@ -409,29 +409,29 @@ export default function Home({ total, today }) {
...
-                          </span>
-                        </div>
-                        <h3 className="mt-8 text-lg font-semibold leading-8 tracking-tight">
-                          <Link
-                            href={feature.path}
-                            className="text-gray-900 hover:text-indigo-600 hover:underline"
-                          >
+                    <Link
+                      href={feature.path}
+                      className="text-gray-900"
```
Comment (Aadarsh805) [inline pages/index.js:414]: Also add this aria-label on the link ` <Link aria-label="Go to ${feature.name} page"`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 2 above)
Comment (Sriparno08) [reply pages/index.js:414]: Alright 👍

File: pages/index.js [hunk 3]
```diff
@@ -409,29 +409,29 @@ export default function Home({ total, today }) {
...
+                        <div className="-mt-6">
+                          <div>
+                            <span className="inline-flex items-center justify-center rounded-xl bg-indigo-500 p-3 shadow-lg">
+                              <feature.icon
+                                className="h-8 w-8 text-white"
+                                aria-hidden="true"
+                              />
+                            </span>
+                          </div>
+                          <h3 className="mt-8 text-lg font-semibold leading-8 tracking-tight">
```
Comment (Aadarsh805) [inline pages/index.js:426]: Keep the hover in h3, but make it happen on group hover like this, add this class on h3 
`group-hover:underline group-hover:text-indigo-600`

File: pages/index.js [hunk 4]
```diff
@@ -409,29 +409,29 @@ export default function Home({ total, today }) {
...
-                          </span>
-                        </div>
-                        <h3 className="mt-8 text-lg font-semibold leading-8 tracking-tight">
-                          <Link
-                            href={feature.path}
-                            className="text-gray-900 hover:text-indigo-600 hover:underline"
-                          >
+                    <Link aria-label="Go to ${feature.name} page"
+                      href={feature.path}
+                      className="text-gray-900 hover:text-indigo-600"
```
Comment (Aadarsh805) [inline pages/index.js:414]: remove the hover from here and just add a `group` class

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 4 above)
Comment (Aadarsh805) [reply pages/index.js:414]: Now with this the hover changes will happen on `h3` but when hovered over the whole card `group`

Comment (Sriparno08) [reply pages/index.js:414]: Okay

File: pages/index.js (hunk 3 above)
Comment (Sriparno08) [reply pages/index.js:426]: Got it!

File: pages/index.js (hunk 4 above)
Comment (eddiejaoude) [reply pages/index.js]: oh I didn't know about that, thank you @Aadarsh805 👍

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍 Great collaboration 🔥

**Comment by Sriparno08** [comment]
Thanks, Eddie!

**Pull request merged**